ios/Podfile

```diff
@@ -52,10 +52,7 @@ target 'Rainbow' do
   pod 'Shimmer'
   use_native_modules!
   if ENV['SKIP_FLIPPER'] != "true"
-    # If you want to use flipper, uncomment it and install pods.
-    # Remember to not commit flipper dependencies in Podfile.lock,
-    # you can do that by commenting below line back and installing pods again
-    # use_flipper!({'Flipper' => '0.162.0'}) # should match the version of your Flipper client app
+    use_flipper!({'Flipper' => '0.162.0'}) # should match the version of your Flipper client app
     pod 'RNExitApp', :path => '../node_modules/react-native-exit-app'
```

Comment:
I added these comments in my previous PR, the skip flipper part is because we want to skip flipper in E2E builds to save time on building the app. Apparently building with Flipper included takes significantly more time.

I added these comments and commented these out because I was thinking it makes sense to not include flipper in Podfile.lock. But I think it doesn't make a difference. So I'm reverting that change and hoping the CI will continue to work fine.

More details about the skip flipper if statement you can get from @brunobar79

Reply:
We discussed with Skylar, and these two likely shouldn't be under this check